app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -108,6 +110,15 @@ class SessionsFragment : Fragment(), Injectable, Findable, OnReselectedListener
                 .of(this, viewModelFactory)
                 .get(SessionsViewModel::class.java)
 
+        if (Prefs.enableReopenPreviousRoomSessions and (savedInstanceState == null)) {
+            when (Prefs.previousSessionTab) {
+                SessionTabMode.SCHEDULE.name ->
+                    sessionsViewModel.changeTabMode(SessionTabMode.SCHEDULE)
```

Comment:
Can we make this one line by enum support? 😄

Reply:
Okay I try it!! 🙇